Automattic/jetpack
# Social: Update media auto-conversion/validation notices in the editor

## Proposed changes:

* Remove auto-conversion notices from the editor
* Add a new media requirements notice

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to post editor
* Upload an invalid media
* Confirm that you see the updated notice


<table>
    <tbody>
        <tr>
            <th>BEFORE</th>
            <th>AFTER</th>
        </tr>
        <tr>
            <td>


![image](https://github.com/user-attachments/assets/0f5a0a06-6cfb-4835-9f7c-feb0e2f017d0)

</td>
            <td>

![image](https://github.com/user-attachments/assets/9210de97-a7af-4602-9d7f-9e5706a8220e)

</td>
        </tr>
    </tbody>
</table>
